src/lzf.cc

```diff
@@ -17,75 +17,70 @@ using namespace node;
 
 
 // Handle<Value> ThrowNodeError(const char* what = NULL) {
-//     return NanThrowError(Exception::Error(NanNew<String>(what)));
+//     return Nan::ThrowError(Exception::Error(Nan::New<String>(what)));
 // }
 NAN_METHOD(compress) {
-    if (args.Length() < 1 || !Buffer::HasInstance(args[0])) {
-        return NanThrowError("First argument must be a Buffer");
+    if (info.Length() < 1 || !Buffer::HasInstance(info[0])) {
+        return Nan::ThrowError("First argument must be a Buffer");
     }
-    NanScope();
 
-    Local<Object> bufferIn = args[0]->ToObject();
```

Comment:
@bobrik Do you remeber, the purpose of convertion of `args[0]` to `Object`? There is `size_t Length(v8::Local<v8::Value> val);` and `char* Data(v8::Local<v8::Value> val);` which should work fine without conversion.

The question is related to the pull request because `Local<Object> Value::ToObject()` method is now deprecated and should be replaced with another call or removed at all.